suite-native/device/src/components/UninitializedDeviceModalAppendix.tsx

```diff
@@ -0,0 +1,88 @@
+import { VStack, HStack, Text, Box } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { Icon, IconName } from '@suite-common/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+import { TxKeyPath } from '@suite-native/intl';
+
+const listItemStyle = prepareNativeStyle(() => ({
+    width: '90%',
+}));
+
+const iconBackgroundStyle = prepareNativeStyle<{ color: Color }>((utils, { color }) => ({
+    backgroundColor: utils.colors[color],
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: utils.spacings.extraLarge,
+    height: utils.spacings.extraLarge,
+    borderWidth: color ? utils.borders.widths.small : 0,
+    borderColor: utils.colors.borderElevation1,
+    borderRadius: 12,
```

Comment:
nit: since there are multiple style props depending on the `color` I suggest to group the in a single condition:


```suggestion
    borderRadius: 12,
    extend: {
        condition: G.isNotNullable(color),
        style: {
            backgroundColor: utils.colors[color],
            borderWidth: utils.borders.widths.small,
            borderColor: utils.colors.borderElevation1,
        },
    },
```

Reply:
nice, thank you for the tip 👍 c50a89424eda2e436bb3840da71c1f713838e8a1